### DISCUSSION TITLE
Add: Mongol TV

### DISCUSSION THREAD

**Discussion by AVGCFan1995** [iptv-org/community#1166] [Show and tell]
### Channel ID (required)

MongolTV.mn

### Channel Name

Mongol TV

### Stream URL (required)

http://110.19.106.198:9981/stream/channelid/149428450?ticket=66738F92816A16E48EBB4DB20E3AC6B5A76622B9&profile=pass

### Quality

None

### Label

None

### Notes

Hey, @BellezaEmporium! I just found the livestream link for Mongol TV but there's a pop-up text that says, “Please enter a valid login name and a password for realm tvheadend.” but how it's going to work?!
![20241222_232528](https://github.com/user-attachments/assets/cc572b3a-d7ee-4299-95bf-9582c5d7a33e)

BTW, here's the logo.
![b83e5673-e763-49c2-abf4-7ca703af66e4](https://github.com/user-attachments/assets/f970bc48-335f-40ba-86e8-f7e20769337f)

### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#how-to-add-a-new-stream-link-to-a-playlists
